src/frontend/src/views/Upload/components/Metadata/components/StaticTable/index.tsx

```diff
@@ -147,44 +145,139 @@ export default function StaticTable({
+              <StyledHeaderTooltip
+                title={
+                  <>
+                    <B>Collection Date:</B> The date the sample was collected
+                    from the host.
+                  </>
+                }
+                placement="bottom"
+                sdsStyle="light"
+                arrow={true}
+              >
+                <StyledTableCell>
+                  {SAMPLE_UPLOAD_METADATA_KEYS_TO_HEADERS.collectionDate}
+                </StyledTableCell>
+              </StyledHeaderTooltip>
+              <StyledHeaderTooltip
+                title={
+                  <>
+                    <B>Collection Location:</B> The location where sample
+                    collection occurred. This field is filled in using the
+                    closest match available in the GISAID database.
+                  </>
+                }
+                placement="bottom"
+                sdsStyle="light"
+                arrow={true}
+              >
+                <StyledTableCell>
+                  {SAMPLE_UPLOAD_METADATA_KEYS_TO_HEADERS.collectionLocation}
+                </StyledTableCell>
+              </StyledHeaderTooltip>
+              <StyledHeaderTooltip
+                title={
+                  <>
+                    <B>Sequencing Date:</B> Date on which the sample was
+                    sequenced.
+                  </>
+                }
+                placement="bottom"
+                sdsStyle="light"
+                arrow={true}
+              >
+                <StyledTableCell>
+                  {SAMPLE_UPLOAD_METADATA_KEYS_TO_HEADERS.sequencingDate}
+                </StyledTableCell>
+              </StyledHeaderTooltip>
+              <StyledHeaderTooltip
+                title={
+                  <>
+                    <B>Sample is Private:</B> If private, your sample will not
+                    be shared beyond your group.
+                  </>
+                }
+                placement="bottom"
+                sdsStyle="light"
+                arrow={true}
+              >
+                <PrivateTableCell align="center">
+                  {SAMPLE_UPLOAD_METADATA_KEYS_TO_HEADERS.keepPrivate}
+                </PrivateTableCell>
+              </StyledHeaderTooltip>
             </StyledTableRow>
           </TableHead>
           {metadata && (
-            <TableBody>
-              {errorSortedMetadata.map(([sampleId, sampleMetadata]) => {
-                return (
-                  <Row
-                    key={sampleId}
-                    id={sampleId}
-                    metadata={sampleMetadata}
-                    validationError={validationErrors[sampleId]}
-                  />
-                );
-              })}
-            </TableBody>
+            <StyledBodyTooltip
+              title={
+                "Import metadata into this table by uploading a TSV or CSV. Download the metadata template above to get started."
+              }
```

Comment:
string props don't need to be wrapped in curly braces

```suggestion
              title="Import metadata into this table by uploading a TSV or CSV. Download the metadata template above to get started."
```